include/h2o/httpclient.h

```diff
@@ -377,6 +377,7 @@ extern const char h2o_httpclient_error_internal[];
 extern const char h2o_httpclient_error_malformed_frame[];
 
 extern const char h2o_httpclient_upgrade_to_connect[];
+extern const char h2o_httpclient_upgrade_to_connect_udp[];
```

Comment:
I'm afraid we cannot define a global pointer that indicates use of "connect-udp," as the upgrade token is sent from the client over the network and we decode them at various places; we do not necessarily convert those tokens to this global pointer.

`h2o_httpclient_upgrade_to_connect` really stands out because it is an internal representation of lack of the upgrade token (which indicates use of traditional CONNECT). Maybe we should add a comment stating as such.